distributionviewer/core/static/js/app/components/containers/chart-list-container.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ChartList from '../views/chart-list';
+
+// numCharts is updated and provided to ChartList just for demonstration
+// purposes, and so that ChartList doesn't just have to include <ExampleChart />
+// 30 times in its source.
+const ChartListContainer = React.createClass({
+  getInitialState: function() {
+    return {
+      numCharts: 0
+    }
+  },
+
+  componentDidMount: function() {
```

Comment:
I'd like to start using ES2015 style arrows going forward since it makes things less verbose especially wrt react. I know I wasn't too consistent on this with Shielddash on this (and it's only a nitpick).